raws-feed-processor/src/main/config/raws-feed-processor.properties

```diff
@@ -0,0 +1,25 @@
+raws.mesowest.api.key=KzCad2ngouvo0itVwDZDOVajikoA3oaVmzq
+raws.mesowest.feed.variables=air_temp,wind_speed,wind_direction,wind_gust,dew_point_temperature,relative_humidity
+raws.mesowest.feed.output.type=geojson
+raws.mesowest.feed.states.of.interest=CA,AZ,NV,OR
+raws.mesowest.feed.units=temp|f,speed|mph
+raws.mesowest.api.url=https://api.synopticlabs.org/v2/
+raws.mesowest.api.get.latest.observations.path=stations/latest?&network=2&within=5&output={outputype}&state={states_of_interest}&units={units}&vars={variables}&token={token}
+# instead of station ids
+raws.mesowest.api.latest.in.last.x.min=https://api.synopticlabs.org/v2/stations/latest?token=cb4aed936feb4b678dc7e704ac927b38&network=2&output=geojson&state=CA,NV,AZ,OR&units=temp|f,speed|mph&vars=air_temp,wind_speed,wind_direction,wind_gust,dew_point_temperature,relative_humidity&obtimezone=local&within=5
+raws.mesowest.api.latest.all=https://api.synopticlabs.org/v2/stations/latest?token=cb4aed936feb4b678dc7e704ac927b38&network=2&output=geojson&state=CA,NV,AZ,OR&units=temp|f,speed|mph&vars=air_temp,wind_speed,wind_direction,wind_gust,dew_point_temperature,relative_humidity&obtimezone=local
+
+raws.mesowest.processor.run.frequency=5m
+
+db.type=postgis
+db.host=postgis
+db.port=5432
+db.name=nics.datafeeds
+db.user=nics
```

Comment:
Also, Irving is checking in these files into puppet repo for staging & prod. We are also working on checking in configs for test & training.